repos/fdbt-site/src/data/auroradb.ts

```diff
@@ -1202,6 +1202,34 @@ export const getGroupPassengerTypesFromGlobalSettings = async (nocCode: string):
     }
 };
 
+export const getPassengerTypeNameByIdAndNoc = async (id: number, nocCode: string): Promise<string> => {
```

Comment:
Does this not exist already? Thought J added something like this? Alternatively you could just use the getPassengerTypeById and just use the name?

Reply:
My bad, I think we hadn't pulled J's changes when we wrote this - will get rid!